_posts/2021-10-06-#23157.md

```diff
@@ -10,15 +10,80 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
+We recently [looked at PR 22677](./22677), which cuts the circular dependency
```

Comment:
I usually put the # sign in front of PR numbers but up to you
```suggestion
We recently [looked at PR #22677](./22677), which cuts the circular dependency
```

Reply:
Done!